.project-keeper.yml

```diff
@@ -2,4 +2,5 @@ sources:
   - type: maven
     path: pom.xml
     modules:
-      - maven_central
\ No newline at end of file
+      - maven_central
+      - integration_tests
```

Comment:
Build already runs with Java 17. No need to add separate `next-java` workflow. Please delete the workflow and add an exclusion here.

Reply:
Wouldn't the right strategy to test with Java 21?